pages/ecosystem.vue

```diff
@@ -251,6 +266,12 @@ function sortMembers(membersToSort: Member[]) {
     ? reverse(membersOnAscOrder)
     : membersOnAscOrder;
 }
+
+function getFilteredResultsCount(tierName: string): number {
```

Comment:
The code that involves this function is repeated and confusing, we can discuss this offline

Reply:
Refactored in https://github.com/Qiskit/qiskit.org/pull/3322/commits/17668d0227c79516aef46dd65c064c5be831eb61